charter-draft.html

```diff
@@ -103,33 +107,47 @@ <h1 id="title">
 				<h2 id="scope">
 					Scope
 				</h2>
-				<p>	The Privacy Interest Group (PING) provides other W3C groups with feedback on
-					(potential) privacy issues in their specifications as well as suggestions for
-					mitigating those issues.
+				<p>	The Privacy Interest Group (PING) develops and documents guidelines,
+					patterns, processes and best practices for addressing privacy
+					considerations in Web standards.
 				</p>
-				<p>	PING also develops and documents guidelines, patterns, processes and best
-					practices for addressing privacy considerations in Web standards.
+				<p>	PING provides "horizontal review" - offering groups developing web standards
+					on-request guidance on privacy issues and mitigations specific to their
+					technologies.  PING aims to offer this review as early in the technology
+					development lifecycle as requested, observing that early feedback is often
+					more helpful.  PING may also seek out technologies that might benefit from
+					earlier privacy review and conduct such reviews on its own initiative.
 				</p>
-				<p>	PING may also incubate recommendation-track standards work
-					on privacy issues by collecting requirements, prototyping, and/or
-					initiating the work within the IG and
-					recommending that the W3C move the work into other groups when
+				<p>	From time to time, PING incubates standards work on privacy issues by
+					collecting requirements, prototyping, and/or initiating the work within
+					the IG and recommending that the W3C move the work into other groups when
 					appropriate.
+				</p>	
+				<p>	When privacy issues are found in existing features of the Web platform,
+					PING may recommend mitigations for those issues,
+					up to and including deprecation of the features.
 				</p>
 				<h3>
 					Success Criteria
 				</h3>
 				<ul>
-					<li>	Timely feedback to other W3C groups, when requested, regarding privacy issues
-						in their specifications.
+					<li>	Feedback to other W3C groups, upon request, regarding privacy issues
+						in their specifications.</li>
 					<li>	Systematizing the privacy review of Web standards.</li>
 				</ul>
 			</div>
 			<div>
 				<h2 id="deliverables">
 					Deliverables
 				</h2>
-				<p>	PING may publish documents consistent with the above scope such as
+				<p>	In conjunction with W3C's <a href="https://www.w3.org/tag">Technical Architecture Group (TAG)</a>
```

Comment:
(A comment, not editorial...)
I'm hoping we can expand this further as we start new initiatives--like the "big ideas"--but for now, these are the concrete docs we can point to, which help.